src/dsp_tools/utils/xmlupload/upload_stashed_resptr_props.py

```diff
@@ -33,179 +31,100 @@ def upload_stashed_resptr_props(
         nonapplied_resptr_props: the resptr props that could not be uploaded
     """
 
+    # XXX: how to handle not found resources. currently nothing happens apart from a warning. # pylint: disable=fixme
```

Comment:
What's the plan with this?

Reply:
we should discuss it in the next meeting and then see how to move on; but I don't want to have the PR waiting that long (but of course I can delete the comment, if you prefer - I just thought it would make it easier to find the place in question again)

Reply:
I'll remove it - should be easy enough to find again :)